### DISCUSSION TITLE
Always save failed taiko (or all) replays

### DISCUSSION THREAD

**Discussion by Doksha** [ppy/osu#20959] [Ideas]
failing in taiko only happens at the end of the map, so whether you fail or not, you have the same score, just different letter grades. Failing should still automatically save the replay (instead of having to click the save button) so you can see progress through a map without the nofail multiplier getting in the way.

This could also apply to the other modes, although then maybe it should be an option in settings for players who dont want to track failed plays.

**Comment by cdwcgt** [comment]
You should be able to save the replay after fail. Why do you need this?

What I can imagine making it meaningful is that you want save replay after **retry** because there is no option to save retry score mow(but retry score also is a fail score)

**Reply by Doksha** [reply]
in case you accidentally retry/exit the map, and just simple convenience. I know for a fact that I will forget to click the save replay button at some point. The question should be why not? The only issue would be players who dont want failed plays to be saved, which is why I suggested it should be an option.
